### PR TITLE
Migrate benchmark to dotnet core 1.0.0

### PR DESCRIPTION
**Pull request by roman-kozachenko**

### CODE REVIEW

File: msgpack.sln [hunk 1]
```diff
@@ -14,22 +13,53 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "msgpack.light", "src\msgpac
...
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "msgpack.light.tests", "tests\msgpack.light.tests\msgpack.light.tests.csproj", "{4DD9051D-548F-4E11-86EA-746F443CEA67}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{B251925D-B5B7-4114-BC5F-E6DF6D622941}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "msgpack.light.benchmark", "src\msgpack.light.benchmark\msgpack.light.benchmark.csproj", "{62C5CC43-A029-4529-BBCA-6D8CC69E6B11}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
```
Comment (aensidhe) [inline msgpack.sln]: We do not need these configurations.

File: src/msgpack.light.benchmark/msgpack.light.benchmark.csproj [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.10.3" />
+    <PackageReference Include="MsgPack.Cli" Version="0.9.0-beta2" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.1-beta1" />
```
Comment (aensidhe) [inline src/msgpack.light.benchmark/msgpack.light.benchmark.csproj]: Why do we use betas here?

File: src/msgpack.light.benchmark/msgpack.light.benchmark.csproj [hunk 3]
```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+
+  <PropertyGroup>
+    <Description>msgpack.light.benchmark Console Application</Description>
+    <Authors>aensidhe</Authors>
+    <TargetFramework>net45</TargetFramework>
```
Comment (aensidhe) [inline src/msgpack.light.benchmark/msgpack.light.benchmark.csproj]: Does it make sense to target net45 only?

File: msgpack.sln (hunk 1 above)
Comment (roman-kozachenko) [reply msgpack.sln]: Removed all except AnyCpu

File: src/msgpack.light.benchmark/msgpack.light.benchmark.csproj (hunk 3 above)
Comment (roman-kozachenko) [reply src/msgpack.light.benchmark/msgpack.light.benchmark.csproj]: netstandard1.1 added (netstandard1.0 is not supported)

File: src/msgpack.light.benchmark/msgpack.light.benchmark.csproj (hunk 2 above)
Comment (roman-kozachenko) [reply src/msgpack.light.benchmark/msgpack.light.benchmark.csproj]: Forgot to untick "Prereleases" checkbox.
Latest stable versions used

**Review by aensidhe — APPROVED** [review]

**Pull request merged**